spec/integration/integration_helper.rb

```diff
@@ -17,6 +17,16 @@
 require 'lhm/sql_helper'
 
 module IntegrationHelper
+
+  def self.included(base)
+    base.after(:each) do
+      cleanup_connection = new_mysql_connection
+      results = cleanup_connection.query("SELECT table_name FROM information_schema.tables WHERE table_schema = '#{$db_name}';")
+      table_names_for_cleanup = results.map { |row| "#{$db_name}." + row.values.first }
+      cleanup_connection.query("DROP TABLE #{table_names_for_cleanup.join(', ')};") if table_names_for_cleanup.length > 0
```

Comment:
`DROP TABLE IF EXISTS` is always the way to go.

Reply:
Added.